packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -353,80 +355,89 @@ class ExpressionEditor extends React.Component<Props, State> {
         );
 
         return (
-            <div>
+            <View>
                 <HeadingSmall>Global Options</HeadingSmall>
 
                 <div className={css(styles.paddedY)}>
                     <LabeledTextField
-                        label="Visible label"
+                        label={
+                            <>
+                                Visible label
+                                <InfoTip>
+                                    Optional visible text; strongly encouraged
```

Comment:
Not sure what's our convention on texts. Are we not doing translations with our editor?

This is probably a preference, it would help to make the code look cleaner if we have these text defined somewhere.

```suggestion
                                <InfoTip>{optionalTextInfo}</InfoTip>
```

Reply:
We don't do localization in the editor so I think leaving this inline is more readable/maintainable than adding the indirection of a variable containing the text.  If we needed to localize, then I'd definitely make changes as you suggest.